components/atom/textarea/src/index.js

```diff
@@ -14,14 +14,10 @@ const TEXTAREA_STATES = {
   ALERT: 'alert'
 }
 
-const DEFAULT_PROPS = {
-  size: SIZES.SHORT,
-  onChange: () => {}
-}
-
 const AtomTextarea = ({
-  onChange = DEFAULT_PROPS.onChange,
```

Comment:
for me it is better to define the defaultProps as default arguments (JS native way) but I don't know why this is the only component in sui world that defines the default props in such way. Ideas?

Reply:
I prefer this one instead of defaultProps.

Reply:
Using default props makes components API more resilient and is kinda more declarative...
So, what else? 😄